lib/prelude.nim

```diff
@@ -17,7 +17,9 @@
 ##
 ## .. code-block:: nim
 ##   import os, strutils, times, parseutils, hashes, tables, sets, sequtils
-##   when not defined(js): import parseopt
+##   when not defined(js) or (NimMajor, NimMinor, NimPatch) >= (1, 5, 1):
```

Comment:
This might be too complex to put in a doc, will take possible alternatives. Should it be (1, 5, 1) or (1, 5)?

Reply:
It is an `if` with 2 conditions, users should be able to read it.  :)

Reply:
I prefer (1,5,1) although in this case it makes no difference since we don't have (1,5,0).

I really wish we can start updating `patch` in devel (1.5) as we were doing in 1.3.x; this makes it easier to track in bug reports (tested in 1.5.7 at hash xxx) and makes it easier to use conditional compilation for people on devel that must work on multiple branches